doc/source/io.rst

```diff
@@ -282,16 +285,16 @@ float_precision : string, default None
   Specifies which converter the C engine should use for floating-point values.
   The options are ``None`` for the ordinary converter, ``high`` for the
   high-precision converter, and ``round_trip`` for the round-trip converter.
-lineterminator : str (length 1), default ``None``
```

Comment:
Can you illustrate with an example?  I don't quite understand what you're saying here.